src/custom/pages/TermsAndConditions/TermsAndConditions.md

```diff
@@ -1,166 +1,462 @@
+    Ethereum based organisation managed by community members. To contact
+    us, please write to us via the
+    GnosisDAO subcategory of forum.gnosis.io. 
+2.  We use open source products and developer materials from Gnosis
+    Limited and affiliated persons and entities (“**Affiliates**”) as
+    well as other third parties to provide a Platform to Gnosis Protocol
+    version 2 (the “**Protocol**”). The provision of the Platform and
+    the Protocol is based solely on our independent actions. 
+
+**WE PROVIDE A PLATFORM TO INTERACTING WITH THE GNOSIS PROTOCOL**
+
+1.  We provide a graphical user interface (the “**Platform**”) to
+    facilitate you interacting with the Protocol. 
+2.  The Protocol was developed by Gnosis Limited and is governed by
+    smart contracts that allow permissionless peer-to-peer trades
+    between users applying a batch auction mechanism and without the
+    need for intermediaries on Ethereum Mainnet, and Ethereum Virtual
+    Machine compatible validation mechanisms. We are not a custodian or
+    a counterparty to any transactions executed by you on the Protocol.
+    We do not support any other service, particularly we do not provide
+    any order matching, guaranteed prices, or similar exchange or
+    trading platform services.
+
+**BY USING OUR PLATFORM, YOU ACCEPT THESE** **TERMS**
+
+1.  These are the general terms and conditions ("**Terms**") that apply
+    to the use of our Platform and the Protocol. 
+2.  By using our Platform, you confirm that you (1) accept and agree to
+    these Terms and that you agree
+    to comply with them. If you do not agree, you must not use our
+    Platform. 
+3.  You are responsible for ensuring that all persons who access or use
+    our Platform through your device or internet connection are aware of
+    these Terms, and that they
+    comply with them.
+4.  We may amend these Terms at our
+    sole discretion. We regularly do so. Every time you wish to use our
+    Platform, please check these Terms
+    to ensure you understand the terms that apply at that time.
+5.  We may terminate or suspend your access to our Platform immediately,
+    without prior notice or liability, if you breach any clause of the
+    Terms. Upon termination of your access, your right to use the
+    Platform will immediately cease. Clauses 7 to 22 and any other term
+    intended so will survive any termination of these Terms.
+6.  You may have been recommended to the Platform by a third party. We
+    shall not be liable for any agreement or terms that may exist
+    between you and the respective third party.
+
+**WHAT YOU REQUIRE TO USE OUR PLATFORM**
+
+Users may use our Platform to interact with the Protocol and buy or sell
+ERC 20 tokens by submitting orders into a batch auction running on the
+Protocol. 
+
+A detailed step-by-step guide on how to use the Platform may be found
+[in the “FAQ” section on the
+Platform](https://cowswap.exchange/#/faq#what-is-cowswap). 
```

Comment:
Make it relative? For example, barn.cowswap.exchange will redirect to cowswap.exchange

Reply:
Good point, will change.

Reply:
Updated now!

Reply:
What do you think about opening this in the same page instead of a new one?